OpenRA.Mods.Common/Traits/AutoCrusher.cs

```diff
@@ -50,20 +51,17 @@ public AutoCrusher(Actor self, AutoCrusherInfo info)
 			moveInfo = self.Info.TraitInfo<IMoveInfo>();
 			nextScanTime = self.World.SharedRandom.Next(Info.MinimumScanTimeInterval, Info.MaximumScanTimeInterval);
 			isAircraft = move is Aircraft;
+			ignoresDisguise = self.Info.HasTraitInfo<IgnoresDisguiseInfo>();
 		}
 
 		void INotifyIdle.TickIdle(Actor self)
 		{
-			if (nextScanTime-- > 0)
+			if (IsTraitDisabled || IsTraitPaused || nextScanTime-- > 0)
```

Comment:
Usually pause does different behaviour. Here there's no difference. Perhaps downgrade to conditional trait?

Reply:
Hm, good point. Fixed. 👍